src/popup/router/components/SidebarMenu.vue

```diff
@@ -95,10 +95,13 @@
 import { mapGetters } from 'vuex';
 import Close from '../../../icons/close.svg?vue-component';
 import Arrow from '../../../icons/arrow-current-color.svg?vue-component';
+import UserAvatar from './UserAvatar'
 
 export default {
-  components: { Close, Arrow },
-  computed: mapGetters(['account', 'activeAccountName']),
+  components: { Close, Arrow, UserAvatar },
+  computed: {
+    ...mapGetters(['account', 'activeAccountName']),
+  },
```

Comment:
This change is not necessary

Reply:
Agree, its readable enough just like it was on a single line.